app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,84 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+        @tweets = Tweet.all
+        erb :'/tweets/tweets'
+    else
+        redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    #load create tweet form
+    if logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect "/login"
+    end
+  end
+
+  get '/tweets/:id' do
+    #individual show tweet page
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :'/tweets/show_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    #load form to edit tweet
+    #find specific tweet from params[:id] to preload form
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?